OrbitQt/main.cpp

```diff
@@ -18,7 +18,9 @@
 #include "orbitmainwindow.h"
 #include "version.h"
 
-ABSL_FLAG(bool, non_release_features, false, "");
+ABSL_FLAG(bool, non_release_features, false,
```

Comment:
I think enable_broken_features is more descriptive :) enable_unreleased_features sounds like there are some hidden features that is working but we are hiding which is not the case.

Reply:
I would suggest "experimental_features" (and adapt the help text accordingly). It might not be 100% accurate for the current case, but makes sense also for in-development features and experiments like the frame pointer validation.

Reply:
experimental implies that they are new and somewhat working which is not the case here is it? I agree that framepointers are probably in that category but features disabled in this PR are different.

Reply:
But do you want to have different flags for experimental vs. "non-release" features? The features here are only different for historical reasons, if they would be developed now and would be incomplete, they would surely be considered experimental, no?

Reply:
I think that this flag should not include experimental features. For those, we usually have a different feature flag for each. This is for example the case of both frame pointer unwinding and frame pointer validation.
I agree that we should get rid of the double negation. Personally, "broken" features feels a bit harsh for a name, so I'm still undecided, but the meaning is closer to what we want to achieve here.

Reply:
The way I see it there are 2 categories:1
1. Features we want users to try out but they are incomplete and maybe buggy.
2. Features that are broken and need to be repaired or thrown away. There is no point in trying them out because they do not work at all.

I would say 1 are experimental 2 are broken. And we probably want 2 flags here.

Reply:
For simplicity I would lump them together with experimental features. I don't see what you are gaining by having a different type of unreleased feature here. Features are either "released" or "experimental" or we should get rid of them (or am I missing something). If you want to have specific flags for other features, that's fine, but should be decided per feature.

Reply:
@dimitry- With that categorization, I would be in favor of calling it "broken_features" to make it obvious. Yet, I find the distinction hard to maintain over time. What if you know an experimental feature is broken (say frame pointer validation)? Do we now move it to "broken_features" and change the flag everywhere? I would keep it simple and call everything that is not released experimental.

Reply:
The point is really that these "features" are very close to "we should get rid of them". But since they were once working on Windows we don't want to just throw away all the code (that's the feeling I got). That's very different from experimental features that are being developed and should not be together.

Reply:
Or what about "stale" features or "obsolete" features?

Reply:
But if we agree on "broken" features I'm completely fine.

Reply:
My expectation is actually that we will fix most of these at some point. Going through the list, a lot of them seem useful, we just need to make them work.

Reply:
Yes, at some point, maybe, maybe not, that's why we're not getting rid of them entirely. Still, there's no somewhat-working implementation nor someone working on them&mdash;in other words, nobody is *experimenting* with them.
Not to mention that it would be confusing to set a flag to enable actually-experimental features and suddenly also enable features that don't work at all.

Reply:
"enable_hidden_features"?

Reply:
I've gone with `enable_stale_features` now, in this very moment I like it for what it is doing.

Reply:
+1

Reply:
Yeah, I think that's pretty good and close to what it's doing.